docs/shared/reference/project-configuration.md

```diff
@@ -278,10 +278,10 @@ You can also express the same configuration using:
 
 ```json
 "build": {
-  "dependsOn": [{ "projects": "dependencies", "target": "build" }]
+  "dependsOn": [{ "projects": "{dependencies}", "target": "build" }]
```

Comment:
Don't update this yet because if people do it in older versions, they will get a warning eventually.